src/omnisharp-http-utils.el

```diff
@@ -0,0 +1,39 @@
+
+(defun omnisharp-get-host ()
+  "Makes sure omnisharp-host is ended by / "
+  (if (string= (substring omnisharp-host -1 ) "/")
+      omnisharp-host
+    (concat omnisharp-host "/")))
+
+(defun omnisharp--get-api-url (api-name)
+  (concat (omnisharp-get-host) api-name))
+
+;;;###autoload
+(defun omnisharp-post-http-message (url &optional params)
+  "Post http request to server. Return result."
+  (let* ((response (omnisharp--submit-request url params))
+         (data (elt response 3)))
+    (unless data
+      (message "Error when sending request"))
+    data))
+
+(defun omnisharp--submit-request (url &optional params)
+  (require 'request)
```

Comment:
Do you think we could benefit from a descriptive error message in case the user doesn't have `request` installed? Just checked and it seems to come bundled with spacemacs at least - but all users might not have it.

I'm worried explaining this feature to new users will get annoying at some point without a good message :smiley: